packages/worker/src/index.ts

```diff
@@ -54,15 +54,23 @@ app.proxy = true
 app.use(handleScimBody)
 app.use(koaBody({ multipart: true }))
 
+// @ts-expect-error - RedisStore is not typed correctly
+let store: RedisStore | undefined
+
 const sessionMiddleware: Middleware = async (ctx: any, next: any) => {
-  const redisClient = await redis.clients.getSessionClient()
+  if (!store) {
+    const redisClient = await redis.clients.getSessionClient()
+    // @ts-expect-error - RedisStore is not typed correctly
+    store = RedisStore({ client: redisClient.client })
```

Comment:
NAB: if you use `redisStore` there is no need for the ts-label. 

https://github.com/Budibase/account-portal/blob/master/packages/server/src/index.ts#L52

Reply:
What you've linked there is identical to what I've got here, we've just used an uppercase R for the identifier name. I'm not sure how you've made it type-check 🤔